benchmark/attention_backend/bench.py

```diff
@@ -0,0 +1,242 @@
+import argparse
```

Comment:
Why not use the code from https://github.com/sgl-project/sglang/blob/main/test/srt/experiment_runner.py? Just need to add support for exporting the results, everything else can be reused.